src/main/java/io/kestra/plugin/ldap/Search.java

```diff
@@ -120,9 +99,8 @@ public class Search extends LdapConnection implements RunnableTask<Search.Output
             SUB -- Indicates that the base entry itself and any subordinate entries (to any depth) should be considered.
             SUBORDINATE_SUBTREE -- Indicates that any subordinate entries (to any depth) below the entry specified by the base DN should be considered, but the base entry itself should not be considered, as described in draft-sermersheim-ldap-subordinate-scope."""
     )
-    @Default
-    @PluginProperty
-    private SearchScope sub = SearchScope.SUB;
+    @Builder.Default
+    private Property<SearchScope> sub = Property.of(SearchScope.SUB);
```

Comment:
Hi @loicmathieu , I have an issue right here when compiling and getting the following error message : `Caused by: java.lang.IllegalArgumentException: Cannot deserialize value of type `java.lang.String` from Object value (token `JsonToken.START_OBJECT`)` 
What is your recommendation in this case ? At the moment I've found a workaround creating an internal enum and then using the `valueOf` method from SearchScope class to convert to the appropriate object.